<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="3" failed="0" ignored="0" total="3" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="4235" started-at="2020-01-31T03:19:50Z" finished-at="2020-01-31T03:19:55Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="4235" started-at="2020-01-31T03:19:50Z" finished-at="2020-01-31T03:19:55Z">
      <class name="orangeHRM.testcases.LoginScenarios">
        <test-method status="PASS" signature="setupReport()[pri:0, instance:orangeHRM.testcases.LoginScenarios@1e88b3c]" name="setupReport" is-config="true" duration-ms="218" started-at="2020-01-31T03:19:50Z" finished-at="2020-01-31T03:19:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupReport -->
        <test-method status="FAIL" signature="setUp()[pri:0, instance:orangeHRM.testcases.LoginScenarios@1e88b3c]" name="setUp" is-config="true" duration-ms="4174" started-at="2020-01-31T03:19:50Z" finished-at="2020-01-31T03:19:55Z">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[null value in entry: url=null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: null value in entry: url=null
at com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:32)
at com.google.common.collect.SingletonImmutableBiMap.<init>(SingletonImmutableBiMap.java:42)
at com.google.common.collect.ImmutableBiMap.of(ImmutableBiMap.java:70)
at com.google.common.collect.ImmutableMap.of(ImmutableMap.java:123)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:273)
at orangeHRM.factories.BrowserFactory.getApplication(BrowserFactory.java:41)
at orangeHRM.pages.BaseClass.setUp(BaseClass.java:92)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="verifyPage()[pri:0, instance:orangeHRM.testcases.LoginScenarios@1e88b3c]" name="verifyPage" duration-ms="0" started-at="2020-01-31T03:19:55Z" finished-at="2020-01-31T03:19:55Z">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[null value in entry: url=null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: null value in entry: url=null
at com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:32)
at com.google.common.collect.SingletonImmutableBiMap.<init>(SingletonImmutableBiMap.java:42)
at com.google.common.collect.ImmutableBiMap.of(ImmutableBiMap.java:70)
at com.google.common.collect.ImmutableMap.of(ImmutableMap.java:123)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:273)
at orangeHRM.factories.BrowserFactory.getApplication(BrowserFactory.java:41)
at orangeHRM.pages.BaseClass.setUp(BaseClass.java:92)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:168)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:105)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyPage -->
        <test-method status="SKIP" signature="tearDownReport(org.testng.ITestResult)[pri:0, instance:orangeHRM.testcases.LoginScenarios@1e88b3c]" name="tearDownReport" is-config="true" duration-ms="0" started-at="2020-01-31T03:19:55Z" finished-at="2020-01-31T03:19:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownReport -->
        <test-method status="SKIP" signature="loginToApplication()[pri:1, instance:orangeHRM.testcases.LoginScenarios@1e88b3c]" name="loginToApplication" duration-ms="0" started-at="2020-01-31T03:19:55Z" depends-on-methods="orangeHRM.testcases.LoginScenarios.verifyPage" finished-at="2020-01-31T03:19:55Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LoginScenarios.loginToApplication()[pri:1, instance:orangeHRM.testcases.LoginScenarios@1e88b3c] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LoginScenarios.loginToApplication()[pri:1, instance:orangeHRM.testcases.LoginScenarios@1e88b3c] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginToApplication -->
        <test-method status="SKIP" signature="logoutFromApplication()[pri:2, instance:orangeHRM.testcases.LoginScenarios@1e88b3c]" name="logoutFromApplication" duration-ms="0" started-at="2020-01-31T03:19:55Z" depends-on-methods="orangeHRM.testcases.LoginScenarios.loginToApplication" finished-at="2020-01-31T03:19:55Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method LoginScenarios.logoutFromApplication()[pri:2, instance:orangeHRM.testcases.LoginScenarios@1e88b3c] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method LoginScenarios.logoutFromApplication()[pri:2, instance:orangeHRM.testcases.LoginScenarios@1e88b3c] depends on not successfully finished methods
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logoutFromApplication -->
        <test-method status="SKIP" signature="tearDown()[pri:0, instance:orangeHRM.testcases.LoginScenarios@1e88b3c]" name="tearDown" is-config="true" duration-ms="0" started-at="2020-01-31T03:19:55Z" finished-at="2020-01-31T03:19:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- orangeHRM.testcases.LoginScenarios -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
